modules/mssqlserver/src/main/java/org/testcontainers/containers/MSSQLServerContainer.java

```diff
@@ -106,12 +112,43 @@ protected String constructUrlForConnection(String queryString) {
         return super.constructUrlForConnection(queryString);
     }
 
+    /**
+     * @return jdbc url for connecting to default database
+     */
     @Override
     public String getJdbcUrl() {
-        String additionalUrlParams = constructUrlParameters(";", ";");
+        return getJdbcUrl(urlParameters);
```

Comment:
Not calling `getJdbcUrl(getDatabaseName()` here, to make sure I don't break anyones code.

If they set `database`/`databaseName` parameter, it has to stay working as it was.